subsys/bluetooth/mesh/Kconfig

```
@@ -52,13 +52,41 @@ config BT_MESH_UNPROV_BEACON_INT
 	  This option specifies the interval (in seconds) at which the
 	  device sends unprovisioned beacon.
 
+if BT_MESH_PB_ADV
+
 config BT_MESH_PB_ADV_RETRANS_TIMEOUT
 	int "Timeout value of retransmit provisioning PDUs"
 	default 500
 	range 100 800
 	help
 	  Timeout value of retransmit provisioning PDUs.
 
+config BT_MESH_PB_ADV_TRANS_PDU_RETRANSMIT_COUNT
+	int "Provision pdu retransmit count"
+	default 0
+	range 0 7
+	help
+	  Controls the number of retransmissions of original transaction pdu,
```

Suggestion to add "Link Open" to the description:
```suggestion
	int "Link Open and Transaction PDU retransmit count"
	default 0
	range 0 7
	help
	  Controls the number of retransmissions of original Link Open and Transaction PDU,
```